src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component<{}, {}> {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', ev => this.handler(ev));
```

Comment:
```suggestion
    document.addEventListener('keyup', event => this.handler(event));
```
use full words in variables name (in the future)

Reply:
okay)